cmd/flags.go

```diff
@@ -153,8 +153,12 @@ func addStrictFlag(fs *pflag.FlagSet, strict *bool, value bool) {
 	fs.BoolVarP(strict, "strict", "S", value, "enable compiler strict mode")
 }
 
-func addRegoV1FlagWithDescription(fs *pflag.FlagSet, regoV1 *bool, value bool, description string) {
+func addRegoV0V1FlagWithDescription(fs *pflag.FlagSet, regoV1 *bool, value bool, description string) {
+	fs.BoolVar(regoV1, "v0-v1", value, description)
```

Comment:
Not sure if `--v0-v1` is the best name, but I can't come up with anything better.

Could call it `--v0-v1-compatible` to be more similar to `--v0-compatible`; but that command covers a slightly different use-case, so could be more confusing than helpful.

Suggestions are welcome :).